src/BtTreeView.cpp

```diff
@@ -351,19 +393,19 @@ void BtTreeView::setupContextMenu(QWidget* top, QWidget* editor)
 
          break;
       case BtTreeModel::EQUIPMASK:
-         _newMenu->addAction(tr("Equipment"), editor, SLOT(newEquipment()));
+         _newMenu->addAction(tr("Equipment"), this, SLOT(newIngredient()));
          break;
       case BtTreeModel::FERMENTMASK:
-         _newMenu->addAction(tr("Fermentable"), editor, SLOT(newFermentable()));
+         _newMenu->addAction(tr("Fermentable"), this, SLOT(newIngredient()));
          break;
       case BtTreeModel::HOPMASK:
-         _newMenu->addAction(tr("Hop"), editor, SLOT(newHop()));
+         _newMenu->addAction(tr("Hop"), this, SLOT(newIngredient()));
          break;
       case BtTreeModel::MISCMASK:
-         _newMenu->addAction(tr("Misc"), editor, SLOT(newMisc()));
+         _newMenu->addAction(tr("Misc"), this, SLOT(newIngredient()));
          break;
       case BtTreeModel::STYLEMASK:
-         _newMenu->addAction(tr("Style"), editor, SLOT(newStyle()));
+         _newMenu->addAction(tr("Style"), editor, SLOT(newIngredient()));
```

Comment:
I've seen a warning about this line:
`QObject::connect: No such slot StyleEditor::newIngredient() in /home/theo/brewtarget_dev/src/BtTreeView.cpp:408
QObject::connect:  (receiver name: 'styleEditor')`

It seems to me that it should be 
`C++
_newMenu->addAction(tr("Style"), editor, SLOT(newStyle()));`

instead of:
`C++
_newMenu->addAction(tr("Style"), editor, SLOT(newIngredient()));`